packages/components/src/angle-picker-control/angle-circle.tsx

```diff
@@ -52,13 +79,12 @@ function AngleCircle( { value, onChange, ...props } ) {
 			}
 			document.body.style.cursor = 'grabbing';
 		} else {
-			document.body.style.cursor = previousCursorValue.current || null;
+			document.body.style.cursor = previousCursorValue.current || '';
```

Comment:
Runtime change: the correct way to rest a CSS prop is via an empty string (TypeScript would otherwise complain)